### PR TITLE
A first statistics plot

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**

### CODE REVIEW

**Comment by ChristianBeilschmidt** [comment]
[Plot im UI](https://trello.com/c/0vLODPsJ/301-plot-im-ui)

File: projects/wave-core-new/src/lib/operators/dialogs/operator-list/operator-list.component.ts [hunk 1]
```diff
@@ -94,36 +95,39 @@ export class OperatorListComponent implements OnInit {
...
+        //         type: BoxPlotType,
+        //         description: 'Box plot your data'
+        //     },
+        {
+            component: StatisticsPlotComponent,
+            type: {
+                NAME: 'Basic Statistics',
+                ICON_URL: OperatorType.createIconDataUrl('Basic Statistics'),
+            },
+            description: 'Get statistics for any layer'
```
Comment (michaelmattig) [inline projects/wave-core-new/src/lib/operators/dialogs/operator-list/operator-list.component.ts:124]: only works for raster layer currently

**Comment by michaelmattig** [comment]
I have the following issue:
1. Add mock raster
2. Add Basic statistics plot
3. Add Basic statistics plot again

The request
> {"id":"2df14cc6-1b19-471f-be7a-af46fe4410eb","plots":[{"name":"Statistics of NDVI Test Raster","workflow":"f5e62c7e-94eb-59f4-a164-0a6a6d4acb48"},"none"]}

fails with 
> {
	"error": "ProjectUpdateFailed",
	"message": "Failed to update the project."
}

**Comment by ChristianBeilschmidt** [comment]
> I have the following issue:
> 
>     1. Add mock raster
> 
>     2. Add Basic statistics plot
> 
>     3. Add Basic statistics plot again
> 
> 
> The request
> 
> > {"id":"2df14cc6-1b19-471f-be7a-af46fe4410eb","plots":[{"name":"Statistics of NDVI Test Raster","workflow":"f5e62c7e-94eb-59f4-a164-0a6a6d4acb48"},"none"]}
> 
> fails with
> 
> > {
> > "error": "ProjectUpdateFailed",
> > "message": "Failed to update the project."
> > }

cf. https://trello.com/c/pwspiPcD/315-bugfix-plot-project-update

File: projects/wave-core-new/src/lib/operators/dialogs/operator-list/operator-list.component.ts (hunk 1 above)
Comment (ChristianBeilschmidt) [reply projects/wave-core-new/src/lib/operators/dialogs/operator-list/operator-list.component.ts]: 6081d01

**Comment by ChristianBeilschmidt** [comment]
> I have the following issue:
> 
>     1. Add mock raster
> 
>     2. Add Basic statistics plot
> 
>     3. Add Basic statistics plot again
> 
> 
> The request
> 
> > {"id":"2df14cc6-1b19-471f-be7a-af46fe4410eb","plots":[{"name":"Statistics of NDVI Test Raster","workflow":"f5e62c7e-94eb-59f4-a164-0a6a6d4acb48"},"none"]}
> 
> fails with
> 
> > {
> > "error": "ProjectUpdateFailed",
> > "message": "Failed to update the project."
> > }


de07ec5

**Comment by michaelmattig** [comment]
the zoom/fullscreen button does nothing. Maybe remove it.

**Comment by ChristianBeilschmidt** [comment]
> the zoom/fullscreen button does nothing. Maybe remove it.


91bc1f2

**Pull request merged**